src/lib/components/loading.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
```

Comment:
I think an attribution to the creator would be nice: http://tobiasahlin.com/spinkit/